dags/openshift_nightlies/config/install/azure/sdn.json

```diff
@@ -6,19 +6,19 @@
     "openshift_network_type": "OpenShiftSDN",
     "openshift_master_vm_size": "Standard_E16s_v3",
     "openshift_worker_vm_size": "Standard_D8s_v3",
-    "openshift_master_root_volume_size": 256,
-    "openshift_worker_root_volume_size": 128,
+    "openshift_master_root_volume_size": 100,
+    "openshift_worker_root_volume_size": 50,
     "openshift_cidr": "10.128.0.0/14",
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_infra_node_vm_size": "Standard_D16s_v3",
+    "openshift_infra_node_vm_size": "Standard_D48s_v3",
```

Comment:
Was this a shrink?

Reply:
Right, not actually a shrink. Just configured the infra vm size to be in pair with other platforms